src/pymovements/dataset/dataset.py

```diff
@@ -1098,3 +1108,13 @@ def _check_gaze_dataframe(self) -> None:
             raise AttributeError('gaze files were not loaded yet. please run load() beforehand')
         if len(self.gaze) == 0:
             raise AttributeError('no files present in gaze attribute')
+
+    def to_yaml(self, yaml_path: str | Path) -> None:
+        """Write the dataset definition to a YAML file.
+
+        Parameters
+        ----------
+        yaml_path: str | Path
+            Path where to save the YAML file to.
+        """
+        self.definition.to_yaml(yaml_path)
```

Comment:
If you don't want to write an additional test you can also remove this function again and we'll add it somewhen in the future. Just as you like.

Reply:
wdym?

Reply:
it's not covered by any test currently, and I think both options are fine

Reply:
ah here I see it, I'll add a test